docs/extend/modular-resources/key-concepts.md

```diff
@@ -0,0 +1,91 @@
+---
+title: "Key Concepts of Modular Resource APIs"
+linkTitle: "Concepts"
+weight: 10
+type: "docs"
+tags: ["server", "rdk", "extending viam", "modular resources", "components", "services"]
+description: "The key concepts behind how Viam's resource APIs and models are uniquely namespaced and modular resource management with the RDK."
+no_list: true
+---
+
+`viam-server` [manages](#management) modular {{< glossary_tooltip term_id="resource" text="resources" >}} configured on your robot the same way it manages resources that are already built into the Robot Development Kit [(RDK)](/internals/rdk/).
+Two key concepts exist across all Viam resources, both built-in and modular, that make this flexible management possible: uniquely namespaced resource [*APIs*](#apis) and [*models*](#models).
+
+If you create your [own module](/extend/modular-resources/create/), you must register any new APIs and models you define in your model with Viam's model registry in the appropriate namespaces to [configure](/extend/modular-resources/configure/) the modular resource on your robot.
+
+### APIs
+
+Every Viam {{< glossary_tooltip term_id="resource" text="resource" >}} subtype exposes an [application programming interface (API)](https://en.wikipedia.org/wiki/API).
+This can be understood as a description of how you can interact with that resource.
+Each API is described through [protocol buffers](https://developers.google.com/protocol-buffers).
+Viam SDKs [expose these APIs](/internals/robot-to-robot-comms/).
+
+#### Namespace
+
+Each Viam resource's API is uniquely namespaced as a colon-delimited-triplet in the form of `namespace:type:subtype`.
+
+For example:
+
+- The API of built-in component [camera](/components/camera/) is `rdk:component:camera`, which exposes methods such as `GetImage()`.
+- The API of built-in service [vision](/services/vision/) is `rdk:service:vision`, which exposes methods such as `GetDetectionsFromCamera()`.
+
+{{% alert title="Tip" color="tip" %}}
+You can see built-in Viam resource APIs in the [Viam GitHub](https://github.com/viamrobotics/api).
+{{% /alert %}}
+
+### Models
+
+{{% alert title="Modules vs. modular resources" color="tip" %}}
```

Comment:
I think actually move this tip up to just above the API heading. This section on models isn't about modules vs modular resources so I think it fits better up there!

Reply:
done!